src/date-picker/date-picker.jsx

```diff
@@ -149,11 +149,14 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: (date) => {
```

Comment:
>  wonder how this shows up in the docs

Yuck! Yeah, good point. I had tried to put it in its own function, but React was having none of it.